charts/growi/templates/deployment.yaml

```
@@ -53,7 +53,7 @@ spec:
         {{- if eq .Values.fileUpload.type "local" }}
           volumeMounts:
           - name: uploads
-            mountPath: "/opt/growi/public/uploads"
+            mountPath: "/data/uploads"
```

```suggestion
            mountPath: /opt/growi/packages/app/public/uploads
```
Does these lines better reflect your intent?

---

No, I don’t think so. 
Because if the application folder is changed due to an upgrade, the data will be lost again.

The uploads folder (i.e. `{WORKDIR}/public/uploads`) is a symbolic link created by `docker-entrypoint.sh`. Its link source is `/data/uploads` in any version.

```
% docker run -it --rm weseek/growi:5.1.10 /bin/bash
node@3532fac63d44:/opt/growi/packages/app$ cat /docker-entrypoint.sh 
#!/bin/sh

set -e

# Support `FILE_UPLOAD=local`
mkdir -p /data/uploads
if [ ! -e "./public/uploads" ]; then
  ln -s /data/uploads ./public/uploads
fi

chown -R node:node /data/uploads
chown -h node:node ./public/uploads

exec gosu node /bin/bash -c "$@"
```

For reference, In [docker-compose.yml](https://github.com/weseek/growi-docker-compose/blob/68adada4a999e5e3214de013fba6c3b78f75e169/docker-compose.yml#L37), the /data folder is mounted.

---

Oh, sorry...
I also confirmed and GROWI docker images have the ability to create such a symbolic link, at least as of 3.6.0.

https://github.com/weseek/growi/commit/81ce59b13af2059706676df4afafa0ca006410ad

I will merge this, and then release.